src/App.js

```diff
@@ -1,26 +1,198 @@
+import { render } from '@testing-library/react';
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
+import visaLogo from './img/visa.png';
+import masterCardLogo from './img/master-card.svg';
+
+
+//////Iteration1//////
+class IdCardCom extends React.Component {
+  render() {
+    return(
+      <div>
+        <span>
+          <div id="left">
+          <img src={this.props.picture}></img>
+          </div>
+          <div id="right">
+            <p><strong>First name: </strong>{this.props.firstName}</p>
+            <p><strong>Last name: </strong>{this.props.lastName}</p>
+            <p><strong>Gender: </strong>{this.props.gender}</p>
+            <p><strong>Height: </strong>{this.props.height}</p>
+            {/* <p><strong>Birth</strong>{this.props.birth}</p> */}
+          </div>
+        </span>
+      </div>
+    )
+  }
+}
+
+
+//////Iteration2//////
+function LangFr(props) {
```

Comment:
You are repeating yourself a lot here. You can refactor your code using a switch statement.